examples/olsconfig.yaml

```diff
@@ -53,6 +53,9 @@ ols_config:
     k8s_cluster_api: "https://api.example.com:6443"
     k8s_ca_cert_path: "/Users/home/ca.crt"
     skip_tls_verification: false
+  user_data_collection:
```

Comment:
since we might find ourselves adding fields under this section for "transcript" collection also, please prefix these fields to make it clear they apply to feedback collection, not transcript collection

either that or name the top level field to "feedback_collection" or something similar.